docs/reference/commandline/service_tasks.md

```diff
@@ -0,0 +1,95 @@
+<!--[metadata]>
++++
+title = "service tasks"
+description = "The service tasks command description and usage"
+keywords = ["service, tasks"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+**Warning:** this command is part of the Swarm management feature introduced in Docker 1.12, and might be subject to non backward-compatible changes.
+
+# service tasks
+
+```Markdown
+Usage:	docker service tasks [OPTIONS] SERVICE
+
+List the tasks of a service
+
+Options:
+  -a, --all            Display all tasks
+  -f, --filter value   Filter output based on conditions provided
+      --help           Print usage
+  -n, --no-resolve     Do not map IDs to Names
+```
+
+Lists the tasks that are running as part of the specified service. This command
+has to be run targeting a Manager node.
+
+
+## Examples
+
+### Listing the tasks that are part of a service
+
+The following command shows all the tasks that are part of the `redis` service:
+
+```bash
+$ docker service tasks redis
+ID                         NAME      SERVICE IMAGE        LAST STATE          DESIRED STATE  NODE
+0qihejybwf1x5vqi8lgzlgnpq  redis.1   redis   redis:3.0.6  RUNNING 8 seconds   RUNNING        manager1
+bk658fpbex0d57cqcwoe3jthu  redis.2   redis   redis:3.0.6  RUNNING 9 seconds   RUNNING        worker2
+5ls5s5fldaqg37s9pwayjecrf  redis.3   redis   redis:3.0.6  RUNNING 9 seconds   RUNNING        worker1
```

Comment:
Not sure why `LAST STATE` / `DESIRED STATE` show in all caps for you. This is supposed to be "title" case:
```
docker service tasks helloworld
ID                         NAME          SERVICE     IMAGE   LAST STATE       DESIRED STATE  NODE
0unc3t7qtto2ras4l2voju2nt  helloworld.1  helloworld  alpine  Running 3 hours  Running        manager1
```